bin/varnishtest/tests/r03012.vtc

```diff
@@ -0,0 +1,24 @@
+varnishtest "vcl.list and cli_limit"
+
+server s1 { }
+
+varnish v1 -vcl+backend { } -start
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+varnish v1 -vcl+backend { }
+
+varnish v1 -expect n_vcl_avail == 10
+
+varnish v1 -cliok "param.set cli_limit 128b"
+
+varnish v1 -clierr 201 "vcl.list"
+
+varnish v1 -cliok "param.set cli_limit 64k"
+
+varnish v1 -clierr 200 "vcl.list"
```

Comment:
Please change `-clierr 200` to `-cliok`.

Reply:
Force pushed the change